includes/fbproduct.php

```diff
@@ -652,8 +725,13 @@ public function prepare_product( $retailer_id = null, $type_to_prepare_for = sel
 			$brand = wp_strip_all_tags( WC_Facebookcommerce_Utils::get_store_name() );
 		}
 
+		$rich_text_description = $this->get_rich_text_description();
+		
+		print_r('source', self::$rich_text_description_source);
```

Comment:
Lets remove the print statement as it is of no use for us as we don't have visibility on locally printed statements and currently we don't have rights to sync/access these logs. Also excessive printing may increase local log file size which may take up most of the storage capacity of hosted servers.
I'm sure there are numerous places in our plugin where we are adding debug logs, but we should understand how can we build process to timely clean up these logs automatically to avoid data storage overload.